Content.Server/Chemistry/EntitySystems/RehydratableSystem.cs

```diff
@@ -30,7 +32,14 @@ private void Expand(EntityUid uid, RehydratableComponent comp)
     {
         _popups.PopupEntity(Loc.GetString("rehydratable-component-expands-message", ("owner", uid)), uid);
 
-        var target = Spawn(comp.TargetPrototype, Transform(uid).Coordinates);
+        var target = new EntityUid();
+        var randomMob = _random.Next(0, 8);
+        string[] randomList = { "MobRatServant", "MobCarpHolo", "MobXenoRavager", "MobAngryBee", "MobAdultSlimesYellowAngry", "MobGiantSpiderAngry", "MobBearSpace", "MobPurpleSnake" };
+
+        if (comp.TargetPrototype == "Random")
+            target = Spawn(randomList[randomMob], Transform(uid).Coordinates);
+        else target = Spawn(comp.TargetPrototype, Transform(uid).Coordinates);
+
```

Comment:
just change TargetPrototype to either be a list of strings or an entity spawn entry. You'll have to update the other instances of it but it'll be pretty easy.

Reply:
Thanks for the tip, i'll try making the change you recommended.